srcpkgs/naken_asm/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'naken_asm'
+pkgname=naken_asm
+version=20190317
```

Comment:
I would just use `2019.03.17` as version number and use `${version/./-}` in `wrksrc` and `disftiles` instead of writing out the version again.